jax/_src/numpy/lax_numpy.py

```diff
@@ -5973,19 +5973,44 @@ def meshgrid(*xi: ArrayLike, copy: bool = True, sparse: bool = False,
   return output
 
 
-@custom_jvp
-@util.implements(np.i0)
```

Comment:
Out of curiosity, why did you move the custom JVP from here to a separate implementation function?

Reply:
The documentation of the API was not updated when I build the docs locally with `custom_jvp` as decorator to `i0`. However when I moved it to a separate implementation function, the document of the API is updated.

Reply:
Maybe it was a cacheing issue? `custom_jvp` should be fine for docs. For example `relu` is decorated with `custom_jvp` and the docs render without an issue: https://jax.readthedocs.io/en/latest/_autosummary/jax.nn.relu.html